### PR TITLE
Section/ Voting Banner

### PR DESCRIPTION
**Pull request by frost-jm**

### CODE REVIEW

File: ui/src/sections/Banners/LandingBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+	const bannerContent = {
+		title: 'You’ve been the top submitter of initiatives in the last 30 days! ',
+		description: 'Keep up the great work! You’ve been contributing greatly to Frost.',
+	};
+	return (
+		<Box
+			sx={{
+				width: '100%',
+				backgroundColor: '#4056C3',
+				padding: '24px ',
```
Comment (Marie-Frost) [inline ui/src/sections/Banners/LandingBanner.tsx:16]: change padding top and bottom

File: ui/src/sections/Banners/LandingBanner.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+			<BannerContent
+				title={bannerContent.title}
+				description={bannerContent.description}
+			/>
+			<img
+				src={Asset}
+				alt='asset'
+				className='banner-asset'
+				style={{
+					height: '80px',
```
Comment (Marie-Frost) [inline ui/src/sections/Banners/LandingBanner.tsx:42]: add width

File: ui/src/sections/Banners/LandingBanner.tsx (hunk 1 above)
Comment (frost-jm) [reply ui/src/sections/Banners/LandingBanner.tsx]: done

File: ui/src/sections/Banners/LandingBanner.tsx (hunk 2 above)
Comment (frost-jm) [reply ui/src/sections/Banners/LandingBanner.tsx:42]: done

**Review by Marie-Frost — APPROVED** [review]

**Pull request merged**